### DISCUSSION TITLE
[@mapbox/geo-viewport]  missing the `allowAntiMeridian` parameter to `viewport()`

### DISCUSSION THREAD

**Discussion by ezheidtmann** [DefinitelyTyped/DefinitelyTyped#66278] [Issues with a @types package]
parameter `allowAntiMeridian` is a boolean parameter, after `allowFloat`.

Package source: https://github.com/mapbox/geo-viewport/blob/master/index.js#L34C15-L34C15
DT type: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mapbox__geo-viewport/index.d.ts#L13

I think it's enough to just add the parameter to index.d.ts

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
Just opened #66301
